components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/main/java/org/wso2/carbon/cluster/coordinator/rdbms/RDBMSCommunicationBusContextImpl.java

```diff
@@ -700,7 +700,7 @@ public void clearMembershipEvents() throws ClusterCoordinationException {
                 boolean isCoordinatorNode = coordinatorNodeId.equals(nodeId);
                 if (resultSet.getBlob(3) != null) {
                     int blobLength = (int) resultSet.getBlob(3).length();
-                    byte[] bytes = resultSet.getBlob(3).getBytes(0L, blobLength);
```

Comment:
It seems we have duplicate code in these two methods. Can we extract a private method to be used in these two places?